osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -544,5 +572,10 @@ private string getParentMessage()
                 return parentComment.HasMessage ? parentComment.Message : parentComment.IsDeleted ? "deleted" : string.Empty;
             }
         }
+
+        public Popover GetPopover()
+        {
+            return new ReportCommentPopover(ReportComment);
+        }
```

Comment:
Use lambda for single-line code like this.
```suggestion
        public Popover GetPopover() => new ReportCommentPopover(ReportComment);
```